front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PipelineTemplateController.java

```diff
@@ -37,29 +37,42 @@
 @RequestMapping("pipelineTemplates")
 public class PipelineTemplateController {
 
-  @Autowired
-  PipelineTemplateDAO pipelineTemplateDAO;
+  @Autowired(required = false)
+  PipelineTemplateDAO pipelineTemplateDAO = null;
 
   // TODO rz - Add fiat authz
 
   @RequestMapping(value = "", method = RequestMethod.GET)
   List<PipelineTemplate> list(@RequestParam(required = false, value = "scopes", defaultValue = "global") List<String> scopes) {
+    if (pipelineTemplateDAO == null) {
```

Comment:
Doesn't look like we have an `@ExceptionHandler` for it but we could add that and kill two birds with one stone.

(otherwise we don't get the exception message in the response)